### PR TITLE
created toUnixTimestamp.ts

### PR DESCRIPTION
**Pull request by caxtonacollins**
this function converts a specified date and time into a Unix timestamp

### CODE REVIEW

**Comment by caxtonacollins** [comment]
@Oshioke-Salaki please review

File: app/utils/toUnixTimestamp.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+function toUnixTimestamp(
+  year: number,
```
Comment (stephanniegb) [inline app/utils/toUnixTimestamp.ts:2]: Please pass them as named arguments

File: app/utils/toUnixTimestamp.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  month: number,
+  day: number,
+  hour: number = 0,
+  minute: number = 0,
+  second: number = 0,
+  timezoneOffset: number = 0
+): number {
+  const date = new Date(Date.UTC(year, month - 1, day, hour, minute, second)); // js Data months are 0-indexed, so i substract 1 from the provided month
```
Comment (stephanniegb) [inline app/utils/toUnixTimestamp.ts:10]: Finally please account for users passing who want to use their local timezone.

**Comment by caxtonacollins** [comment]
@Oshioke-Salaki please review, this should close issue #70.

**Comment by stephanniegb** [comment]
Thanks for your contribution!✨

**Comment by caxtonacollins** [comment]
@Oshioke-Salaki thanks. looking forward to contributing more

**Pull request merged**